homeassistant/core.py

```diff
@@ -608,6 +608,10 @@ def __eq__(self, other):
                 self.state == other.state and
                 self.attributes == other.attributes)
 
+    def __lt__(self, other):
```

Comment:
My primary operation is to insert an item into an already sorted list - but bisect doesn't support passing in a lambda to use as a key AFAIK.

It felt wrong to put the item on the end and then sort the list, but I can do that to avoid needing the __lt__ on the State.